project/ScalafixBuild.scala

```diff
@@ -135,32 +164,20 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     updateOptions := updateOptions.value.withCachedResolution(true),
     ThisBuild / watchTriggeredMessage := Watch.clearScreenOnTrigger,
     commands += Command.command("save-expect") { s =>
-      "unit2_13Target2_13/test:runMain scalafix.tests.util.SaveExpect" ::
-        "unit3Target3/test:runMain scalafix.tests.util.SaveExpect" ::
+      "integration2_13/test:runMain scalafix.tests.util.SaveExpect" ::
+        "integration3/test:runMain scalafix.tests.util.SaveExpect" ::
         s
     },
-    commands += Command.command("ci-3") { s =>
-      "unit2_12Target3/test" ::
-        "unit3Target3/test" ::
+    commands += Command.command("ci-windows") { s =>
+      "publishLocalTransitive" :: // scalafix.tests.interfaces.ScalafixSuite
+        "testOnly -- -l scalafix.internal.tests.utils.SkipWindows" ::
         s
     },
-    commands += Command.command("ci-213") { s =>
-      "unit2_13Target2_13/test" ::
-        "docs2_13/run" ::
+    commands += Command.command("ci-docs") { s =>
+      "docs2_13/run" :: // reduce risk of errors on deploy-website.yml
         "interfaces/doc" ::
-        testRulesAgainstPreviousScalaVersions(scala213, s)
-    },
-    commands += Command.command("ci-212") { s =>
-      "unit2_12Target2_12/test" ::
-        testRulesAgainstPreviousScalaVersions(scala212, s)
-    },
-    commands += Command.command("ci-213-windows") { s =>
-      "publishLocalTransitive" :: // scalafix.tests.interfaces.ScalafixSuite
-        s"unit2_13Target2_13/testOnly -- -l scalafix.internal.tests.utils.SkipWindows" ::
         s
     },
-    // There is flakyness in CliGitDiffTests and CliSemanticTests
-    Test / parallelExecution := false,
```

Comment:
I only observed flakiness in `integration` (expected because we have side effects), so I disabled parallel execution there specifically